content/tracing/setup/ruby.md

```diff
@@ -184,40 +113,51 @@ Ruby APM includes support for the following Ruby interpreters:
-| Resque         | `resque`        | `>= 1.0, < 2.0`        | *[Link][42]*     | *[Link][43]* |
-| Sequel         | `sequel`        | `>= 3.41`              | *[Link][44]*     | *[Link][45]* |
-| Sidekiq        | `sidekiq`       | `>= 4.0`               | *[Link][46]*     | *[Link][47]* |
-| Sinatra        | `sinatra`       | `>= 1.4.5`             | *[Link][48]*     | *[Link][49]* |
-| Sucker Punch   | `sucker_punch`  | `>= 2.0`               | *[Link][50]*     | *[Link][51]* |
+| Name                 | Versions Supported     | Support type    | How to configure |
+| -------------------- | ---------------------- | --------------- | ---------------- |
+| [Active Record][11]  | `>= 3.2, < 5.2`        | Fully supported | *[Link][10]*     |
+| [AWS][13]            | `>= 2.0`               | Fully supported | *[Link][12]*     |
+| [Dalli][15]          | `>= 2.7`               | Fully supported | *[Link][14]*     |
+| [DelayedJob][53]     | `>= 4.1`               | Fully supported | *[Link][52]*     |
+| [Elastic Search][17] | `>= 6.0`               | Fully supported | *[Link][16]*     |
+| [Excon][19]          | `>= 0.62`              | Fully supported | *[Link][18]*     |
+| [Faraday][21]        | `>= 0.14`              | Fully supported | *[Link][20]*     |
+| [gRPC][23]           | `>= 1.10`              | Fully supported | *[Link][22]*     |
+| [Grape][25]          | `>= 1.0`               | Fully supported | *[Link][24]*     |
+| [GraphQL][27]        | `>= 1.7.9`             | Fully supported | *[Link][26]*     |
+| [MongoDB][29]        | `>= 2.0, < 2.5`        | Fully supported | *[Link][28]*     |
+| [MySQL2][55]         | `>= 0.5`               | Fully supported | *[Link][54]*     |
+| [Net/HTTP][31]       | *(Any supported Ruby)* | Fully supported | *[Link][30]*     |
+| [Racecar][33]        | `>= 0.3.5`             | Fully supported | *[Link][32]*     |
+| [Rack][35]           | `>= 1.4.7`             | Fully supported | *[Link][34]*     |
+| [Rails][37]          | `>= 3.2, < 5.2`        | Fully supported | *[Link][36]*     |
+| [Rake][39]           | `>= 12.0`              | Fully supported | *[Link][38]*     |
+| [Redis][41]          | `>= 3.2, < 4.0`        | Fully supported | *[Link][40]*     |
+| [Resque][43]         | `>= 1.0, < 2.0`        | Fully supported | *[Link][42]*     |
+| [RestClient][57]     | `>= 1.8`               | Fully supported | *[Link][56]*     |
+| [Sequel][45]         | `>= 3.41`              | Fully supported | *[Link][44]*     |
+| [Sidekiq][47]        | `>= 4.0`               | Fully supported | *[Link][46]*     |
+| [Sinatra][49]        | `>= 1.4.5`             | Fully supported | *[Link][48]*     |
+| [Sucker Punch][51]   | `>= 2.0`               | Fully supported | *[Link][50]*     |
+
+*Fully supported* support indicates all tracer features are available.
+
+*Experimental* indicates most features should be available, but unverified.
```

Comment:
I don't see this list in the preview link. Where is rendered? Because without this list, people don't know how to enable it via `Datadog.configure`.

Reply:
Resolved (local JS issue).